engine/src/tangl/story/fabula/world.py

```
@@ -555,84 +320,99 @@ def _create_story_full(self, story_label: str) -> StoryGraph:
+
+        return list(
+            self.script_manager.template_factory.find_all(declares_instance=True)
+        )
+
+    def _get_start_address(self) -> str:
+        """Return the address for the starting node."""
+
+        metadata = self.script_manager.get_story_metadata()
+        start_at = metadata.get("start_at")
+        if not start_at:
+            raise ValueError("No start_at specified in metadata")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)